app/api/webhooks/seven-days-passed/route.ts

```
@@ -0,0 +1,3 @@
+export async function POST(request: Request) {
```

ESLint

[error] 'request' is defined but never used.

(@typescript-eslint/no-unused-vars)